src/runtime/api-channel.ts

```diff
@@ -600,6 +602,10 @@ export abstract class PECInnerPort extends APIPort {
   ProxyMessage(@Mapped handle: StorageProxy,
                @Direct message: ProxyMessage<CRDTTypeRecord>,
                @LocalMapped callback: Consumer<Promise<boolean>>): void  {}
+  BackingProxyMessage(@Mapped handle: StorageProxy,
+                 @Direct message: ProxyMessage<CRDTTypeRecord>,
+                 @Direct entityId: string,
+                 @LocalMapped callback: Consumer<Promise<boolean>>): void{}
```

Comment:
Indenting seems a bit off.

Also, I believe that indenting up to the paren has fallen out of favour. I'd suggest:

```
BackingProxyMessage(
    @Mapped handle: StorageProxy,
    @Direct message: ProxyMessage<CRDTTypeRecord>,
    @Direct entityId: string,
    @LocalMapped callback: Consumer<Promise<boolean>>): void {}
```

Additionally, you need a space after `void` (and you may as well delete the extra space after `void` in `ProxyMessage` above)